src/commands/lambda/README.md

```diff
@@ -145,6 +145,34 @@ Instead of supplying arguments, you can create a configuration file in your proj
     }
 }
 ```
+
+## Troubleshooting
```

Comment:
This is misleading, since users might think that this is `Troubleshooting` for the `datadog-ci lambda` commands.

Reply:
I agree, do you have any better suggestions?
Maybe something like 'Getting Support for Datadog Monitoring'?